### PR TITLE
[shopsys] Release of version 13.0.0

### PR DESCRIPTION
**Pull request by grossmannmartin**
| Q             | A
| ------------- | ---
|Description, reason for the PR| New version is here 🎉 
|New feature| No 
|[BC breaks](https://docs.shopsys.com/en/latest/contributing/backward-compatibility-promise/)| No 
|Fixes issues| ... 
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

----
:globe_with_meridians: Live Preview:
  - https://rc-13-0-0.odin.shopsys.cloud
  - https://cz.rc-13-0-0.odin.shopsys.cloud

### CODE REVIEW

File: CHANGELOG-13.0.md [hunk 1]
```diff
@@ -30,3 +30,197 @@ You can see all the changes done to package that you carry about with this tree.
...
+
+## [v13.0.0](https://github.com/shopsys/shopsys/compare/v12.0.0...v13.0.0) (2023-10-09)
+
+### :construction: Changes that require additional implementation if you are using Frontend API
+* Implemented SEO category functionality on storefront by @sebaholesz in https://github.com/shopsys/shopsys/pull/2654
+* improvements to Urql client and query error handling by @sebaholesz in https://github.com/shopsys/shopsys/pull/2659
+* [shopsys] graphql query is not in transaction by @stanoMilan in https://github.com/shopsys/shopsys/pull/2516
+
+### :sparkles: Enhancements and features
+* [shopsys] added various features by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2622
```
Comment (TomasLudvik) [inline CHANGELOG-13.0.md]: I think like we should provide little more detail

File: CHANGELOG-13.0.md [hunk 2]
```diff
@@ -30,3 +30,197 @@ You can see all the changes done to package that you carry about with this tree.
...
+* [shopsys] added detailed opening hours to stores by @TomasLudvik in https://github.com/shopsys/shopsys/pull/2660
+* [shopsys] wishlist by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2640
+* [shopsys] added closing days to be set for stores to inform customers that store is closed by @TomasLudvik in https://github.com/shopsys/shopsys/pull/2685
+* [project-base] restored feeds by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2718
```
Comment (TomasLudvik) [inline CHANGELOG-13.0.md]: I feel like this should not be here

File: CHANGELOG-13.0.md [hunk 3]
```diff
@@ -30,3 +30,197 @@ You can see all the changes done to package that you carry about with this tree.
...
+* [project-base] fixed GetOrdersAsAuthenticatedCustomerUserTest by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2805
+* [shopsys] removed dependency on the graphql from framework by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2809
+* [project-base] order is updated in GoPay only if configuration is set by @TomasLudvik in https://github.com/shopsys/shopsys/pull/2806
+* [shopsys] prevent FileNotFoundException on multiple flush fileupload by @pk16011990 in https://github.com/shopsys/shopsys/pull/2655
+* [project-base] fixed argument passing to productsByCatnum query by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2842
+* [project-base] fixed closed day edit by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2841
+* [project-base] fixed edit language constant by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2845
+* [project-base] added validation to suppliers delivery time by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2843
+* [project-base] fixed sentry logging on production by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2846
```
Comment (TomasLudvik) [inline CHANGELOG-13.0.md]: I would remove this

File: CHANGELOG-13.0.md [hunk 4]
```diff
@@ -30,3 +30,197 @@ You can see all the changes done to package that you carry about with this tree.
...
+* [shopsys] united language constants to English language by @TomasLudvik in https://github.com/shopsys/shopsys/pull/2672
+* [project-base] changed robots.txt datafixture to avoid encouraging inappropriate practices by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2714
+* [project-base] cron modules review by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2733
+* [framework] tests are now multilingual by @TomasLudvik in https://github.com/shopsys/shopsys/pull/2742
+* [shopsys] removed deprecations before release 12.0 by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2768
+* [shopsys] introduced php-fpm base image by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2762
+* [project-base] reduced image url redis cache size by @TomasLudvik in https://github.com/shopsys/shopsys/pull/2761
+* [shopsys] removed unused topMenu article placement by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2776
+* [shopsys] doctrine/orm 2.16.0 and higher are now marked as conflicting by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2691
```
Comment (TomasLudvik) [inline CHANGELOG-13.0.md]: This is not right

File: CHANGELOG-13.0.md [hunk 5]
```diff
@@ -30,3 +30,197 @@ You can see all the changes done to package that you carry about with this tree.
...
+* [shopsys] upgraded doctrine/orm to latest version by @TomasLudvik in https://github.com/shopsys/shopsys/pull/2775
+* [shopsys] updated overblog/graphql-bundle to stable version 1.0.0 with dependencies by @TomasLudvik in https://github.com/shopsys/shopsys/pull/2787
+* [framework] added optional option to export data to Elasticsearch only for the specified domain by @TomasLudvik in https://github.com/shopsys/shopsys/pull/2780
+* [project-base] removed unused code by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2777
+* [shopsys] changed default db server in adminer in local environment by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2803
+* [shopsys] graphql-bundle classes are now dumped so they can use composer autoload by @TomasLudvik in https://github.com/shopsys/shopsys/pull/2830
+* [infrastructure] added information about GitHub limits and increased verbosity of composer install on CI servers by @TomasLudvik in https://github.com/shopsys/shopsys/pull/2860
```
Comment (TomasLudvik) [inline CHANGELOG-13.0.md]: I would remove this

File: CHANGELOG-13.0.md [hunk 6]
```diff
@@ -30,3 +30,197 @@ You can see all the changes done to package that you carry about with this tree.
...
+* [framework] added optional manual readable frequency to crons.yaml by @TomasLudvik in https://github.com/shopsys/shopsys/pull/2854
+
+### :book: Documentation
+* [shopsys] replaced Shopsys Framework with Shopsys Platform by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2682
+* [shopsys] updated open source license acknowledgements by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2709
+* [project-base] minor fix in Readme file by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2737
+* [docs] updated infrastructure schema by @TomasLudvik in https://github.com/shopsys/shopsys/pull/2743
+* [docs] updated PHPStorm settings to use absolute paths for TypeScript by @TomasLudvik in https://github.com/shopsys/shopsys/pull/2760
+* [docs] fixed build on readthedocs by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2811
```
Comment (TomasLudvik) [inline CHANGELOG-13.0.md]: I would remove this

File: CHANGELOG-13.0.md [hunk 7]
```diff
@@ -30,3 +30,197 @@ You can see all the changes done to package that you carry about with this tree.
...
+* [shopsys] added general upgrade notes to upgrade to Shopsys Platform by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2817
+
+### :art: Design & appearance
+* [project-base] product videos UX fixes by @TomasLudvik in https://github.com/shopsys/shopsys/pull/2746
+
+### :cloud: Infrastructure
+* [github-actions] updated GitHub actions for new storefront by @TomasLudvik in https://github.com/shopsys/shopsys/pull/2638
```
Comment (TomasLudvik) [inline CHANGELOG-13.0.md]: I do not think that it is necessary to mention GitHub actions and Gitlab changes and deployment

File: CHANGELOG-13.0.md [hunk 8]
```diff
@@ -30,3 +30,197 @@ You can see all the changes done to package that you carry about with this tree.
...
+* fix console warnings by @tvikito in https://github.com/shopsys/shopsys/pull/2652
+* update of core storefront packages by @sebaholesz in https://github.com/shopsys/shopsys/pull/2665
+* storefront unit tests config and initial examples by @sebaholesz in https://github.com/shopsys/shopsys/pull/2686
+* storefront restructuring by @sebaholesz in https://github.com/shopsys/shopsys/pull/2717
+* removed useTypedTranslationFunction from storefront by @sebaholesz in https://github.com/shopsys/shopsys/pull/2736
+* changed images and icons to free variants by @sebaholesz in https://github.com/shopsys/shopsys/pull/2698
+* added total price to order detail by @sebaholesz in https://github.com/shopsys/shopsys/pull/2702
+
+## New Contributors
+* @tvikito made their first contribution in https://github.com/shopsys/shopsys/pull/2652
```
Comment (TomasLudvik) [inline CHANGELOG-13.0.md]: I am not sure if it is ok to mention core team members

**Review by TomasLudvik — CHANGES_REQUESTED** [review]

File: CHANGELOG-13.0.md (hunk 4 above)
Comment (grossmannmartin) [reply CHANGELOG-13.0.md]: removed

File: CHANGELOG-13.0.md (hunk 7 above)
Comment (grossmannmartin) [reply CHANGELOG-13.0.md]: removed

File: CHANGELOG-13.0.md (hunk 1 above)
Comment (grossmannmartin) [reply CHANGELOG-13.0.md]: I have updated the texts after consultation with @pk16011990

**Review by TomasLudvik — APPROVED** [review]

**Review by pk16011990 — CHANGES_REQUESTED** [review]
1) Missing translation in https://github.com/shopsys/shopsys/blob/2334cf9335095eca8355b812d55902b98eff032c/project-base/storefront/public/locales/cs/common.json#L322

2) Some translations in https://github.com/shopsys/shopsys/blob/2334cf9335095eca8355b812d55902b98eff032c/project-base/storefront/public/locales/cs/common.json are missing in https://github.com/shopsys/shopsys/blob/2334cf9335095eca8355b812d55902b98eff032c/project-base/storefront/public/locales/en/common.json Maybe it is not problem, but It looks weird 🤷

**Pull request merged**